mirdata/utils.py

```diff
@@ -257,3 +260,28 @@ def load_json_index(filename):
     CWD = os.path.dirname(os.path.realpath(__file__))
     with open(os.path.join(CWD, 'indexes', filename)) as f:
         return json.load(f)
+
+
+def check_validated(annotations_path):
+    return os.path.exists(os.path.join(annotations_path, VALIDATED_FILE_NAME))
+
+
+def create_validated(annotations_path):
+    open(os.path.join(annotations_path, VALIDATED_FILE_NAME), 'a').close()
+
+
+def create_invalid(annotations_path, missing_files, invalid_checksums):
+    with open(os.path.join(annotations_path, INVALID_FILE_NAME), 'w') as f:
+        json.dump({'missing_files': missing_files,
+                   'invalid_checksums': invalid_checksums}, f, indent=2)
+
+
+def clobber_all(remote, annotations_path, data_home=None):
```

Comment:
Nit - `annotations_path` --> `downloaded_content_path` or something similar. This should basically be the unzipped file contents right?

Reply:
👍